code/modules/client/client defines.dm

```diff
@@ -17,7 +17,7 @@
 	var/datum/preferences/prefs = null
 	var/move_delay		= 1
 	var/moving			= null
-	var/adminobs		= null

```

Comment:
the var was only used in life() and the code was terrible. The vision refactor makes it useless.